### PR TITLE
Add attribute empty state image when no attribute has been added

### LINKED ISSUE #39680: [Enhancement] Add an empty state illustration to the Attributes block
We can make attributes more visible and easier to understand by adding an illustration to its empty state. It'd only be shown when there are no attributes added.

**Design**

AkVNGImLgSqCObTQ3idVn7-fi-2793_283564

image

**Acceptance criteria**

- [ ] When there are no attributes, we show an empty state illustration in the Attributes block.
- [ ] The illustration is a SVG file with no text strings.
- [ ] The illustration is not shown on mobile.

### PR DESCRIPTION
**Pull request by nathanss**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Add attribute empty state image when no attribute has been added

Closes #39680 .

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the new product editor at /wp-admin/admin.php?page=wc-settings&tab=advanced&section=features
2. Go to Product > Add New
3. Go to the Organization tab
4. Check that there's a drawing (empty state) in the Attributes section
5. Make the viewport's width small (mobile)
6. Check that the empty state disappears
7. Return the viewport's width to normal
8. Create an attribute
9. Check that the empty state disappears
10. Delete the attribute
11. Check that it reappears

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

**Review by mattsherman — CHANGES_REQUESTED** [review]
This implementation has a few issues that I think should be addressed:

- The image has untranslatable text in it -- putting text in an image should be avoided
- The image has hardcoded colors and fonts, etc. in it, that may not match the editor if the editor's styles are updated (either by us or by an extension)

I think a more robust implementation would either:

- Not use an image (I don't know if we have tooling in place to translate strings in SVGs), but rather render this placeholder in React
- Use an image that does not include text

/cc @jarekmorawski (I would have mentioned these potential issues earlier if I had seen/discussed the issue)

**Comment by nathanss** [comment]
I think replacing the English text with "placeholders" is the way to go. 

Also, I checked and it seems we are rendering with React in other places (ProductPageSkeleton), so I will work on converting this to do this as well.

**Comment by nathanss** [comment]
@mattsherman I created a new version:

image

I can also easily add translatable text now. Let's wait for Jarek's call.

File: packages/js/product-editor/src/components/attribute-control/attribute-empty-state-skeleton.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ */
+import { createElement } from '@wordpress/element';
+import classNames from 'classnames';
+
+export function AttributeEmptyStateSkeleton() {
+	return (
+		<div className="woocommerce-product-page-attribute-skeleton">
+			{ Array( 3 )
+				.fill( 0 )
+				.map( ( _, index ) => (
```
Comment (mattsherman) [inline packages/js/product-editor/src/components/attribute-control/attribute-empty-state-skeleton.tsx:12]: Interesting technique... I've learned something new today!

File: packages/js/product-editor/src/components/attribute-control/attribute-skeleton.scss [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	}
+	&__row0 {
+		background-color: $gray-200;
+	}
+	&__row1 {
+		background-color: #e9e9e9;
+	}
+	&__row2 {
+		background-color: #efefef;
+	}
```
Comment (mattsherman) [inline packages/js/product-editor/src/components/attribute-control/attribute-skeleton.scss:32]: You could handle this without separate classnames for each row, and instead use the `:nth-child()` CSS selector, but either approach works!

https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child

**Review by mattsherman — APPROVED** [review]
Nice work changing the implementation to use React components instead of an SVG... much nicer!

**Comment by jarekmorawski** [comment]
Matt, your concerns are valid and I should've mentioned in the original issue that the text strings should be translatable. I'm not sure why I missed that and described the empty state as a static image.

Nathan's updated placeholder is good enough for the MVP but later on, we should have real text. Without it, the empty state doesn't offer much context. My intention was to present examples of attributes the user can (e.g., material) in a tangible way. I'll create a follow-up issue to tackle this at a later date.

File: packages/js/product-editor/src/components/attribute-control/attribute-empty-state-skeleton.tsx (hunk 1 above)
Comment (nathanss) [reply packages/js/product-editor/src/components/attribute-control/attribute-empty-state-skeleton.tsx:12]: Copied from Maikel 😆

File: packages/js/product-editor/src/components/attribute-control/attribute-skeleton.scss (hunk 2 above)
Comment (nathanss) [reply packages/js/product-editor/src/components/attribute-control/attribute-skeleton.scss:32]: oh! I will use this next time. It looks nicer. Thanks for the tip.

**Pull request merged**